packages/eas-json/src/EasJsonSchema.ts

```diff
@@ -102,6 +102,9 @@ const schemaBuildProfileMap: Record<string, Record<string, Joi.Schema>> = {
 };
 
 const EasJsonSchema = Joi.object({
+  experimental: Joi.object({
+    disableBundleIdValidation: Joi.boolean(),
```

Comment:
```suggestion
    disableIosBundleIdValidation: Joi.boolean(),
```
Maybe this is a better name to indicate this is for iOS.